### PR TITLE
[DOCFIX]add unit test for alluxio.cli.fs.command.CopyToLocalCommand in shell …

### PR DESCRIPTION
**Pull request by jackshaw**
finish adding unit test for alluxio.cli.fs.command.CopyToLocalCommand in shell module

### CODE REVIEW

**Comment by jackshaw** [comment]
alluxio-bot, check this please

**Comment by jackshaw** [comment]
alluxio-bot, check this please

**Comment by jackshaw** [comment]
alluxio-bot, check this please

**Comment by jackshaw** [comment]
alluxio-bot, check this please

**Comment by jackshaw** [comment]
alluxio-bot, check this please

**Comment by jackshaw** [comment]
alluxio-bot, check this please

**Comment by jackshaw** [comment]
alluxio-bot, check this please

**Comment by jackshaw** [comment]
alluxio-bot, check this please

**Comment by jackshaw** [comment]
alluxio-bot, check this please

**Comment by jackshaw** [comment]
alluxio-bot, check this please

**Comment by jackshaw** [comment]
alluxio-bot, check this please

**Comment by jackshaw** [comment]
alluxio-bot, check this please

**Comment by jackshaw** [comment]
alluxio-bot, check this please

**Comment by jackshaw** [comment]
alluxio-bot, check this please

**Comment by jackshaw** [comment]
alluxio-bot, check this please

**Comment by jackshaw** [comment]
alluxio-bot, check this please

**Comment by jackshaw** [comment]
alluxio-bot, check this please

**Comment by jackshaw** [comment]
alluxio-bot, check this please

**Comment by jackshaw** [comment]
'alluxio-bot, check this please

**Comment by jackshaw** [comment]
alluxio-bot, check this please

**Comment by jackshaw** [comment]
alluxio-bot, check this please

**Comment by jackshaw** [comment]
alluxio-bot, check this please

**Comment by jackshaw** [comment]
alluxio-bot, check this please

File: shell/src/test/java/alluxio/cli/fs/command/CopyToLocalCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+
+/**
+ * A unit test.
```
Comment (yupeng9) [inline shell/src/test/java/alluxio/cli/fs/command/CopyToLocalCommandTest.java]: Can you make this more descriptive, like `A unit test for @link{CopyToLocalCommand}`.

File: shell/src/test/java/alluxio/cli/fs/command/CopyToLocalCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+ */
+public class CopyToLocalCommandTest {
+  private ByteArrayOutputStream mOutput = new ByteArrayOutputStream();
+  private ByteArrayOutputStream mError = new ByteArrayOutputStream();
```
Comment (yupeng9) [inline shell/src/test/java/alluxio/cli/fs/command/CopyToLocalCommandTest.java]: Are these fields used?

**Pull request closed without merging**